packages/uswds-core/src/styles/mixins/utilities/_disabled.scss

```diff
@@ -0,0 +1,6 @@
+@use "../../functions/utilities/color" as *;
+
```

Comment:
Can you add a comment with a brief description of what this mixin does?

Reply:
Resolved in https://github.com/uswds/uswds/pull/5004/commits/7963934845140e22cc671938cba057fcd5eab31d

Reply:
Sorry - I meant it would be helpful to have description for what `@mixin u-disabled` does. I'll put my comment in a better place next time!

Reply:
GOSH hindsight is 20/20 I feel like a goober for not getting that the first time 😅

Resolved in https://github.com/uswds/uswds/pull/5004/commits/25f19505f450e6a797431c50c684ebffc5979ea0 !